cmd/trace-agent/test/testsuite/cards_test.go

```diff
@@ -73,7 +73,7 @@ apm_config:
 		{
 			conf:    []byte(""),
 			out:     "4166 6766 6766 6746",
-			version: "v0.6",
+			version: "v0.7",
```

Comment:
Why do we need to change this?

Reply:
The version of that endpoint was renamed from v0.6 to v0.7 (as part of the same release). Because the integration tests weren't running (not sure why), these tests weren't failing when they should have, so we didn't notice.